analyze/analyze.rs

```diff
@@ -323,7 +330,10 @@ impl traits::Emit for Paths {
 
             seen.insert(id);
             for caller in items.predecessors(id) {
-                recursive_callers(items, seen, table, depth + 1, caller);
+                if depth != 0 {
```

Comment:
This is almost, but not quite what we want. We want to increment the count whenever there is a *new* branch in the path:

```rust
for (i, caller) in items.predecessors(id) {
    if i > 0 {
        *paths += 1;
    }
    recursive_callers(...);
}
```